main/models.py

```diff
@@ -419,7 +427,9 @@ class Meta:
 
     def __str__(self) -> str:
         """String representation of the Funding object."""
-        return f"{self.project} - £{self.budget:.2f} - {self.project_code}"
+        return (
+            f"{self.project} - £{self.budget:.2f} - {self.cost_centre}_{self.activity}"
```

Comment:
You can still `project_code` as you have added it as a property:
```suggestion
            f"{self.project} - £{self.budget:.2f} - {self.project_code}"
```